ui/components/search/search.js

```diff
@@ -18,21 +27,41 @@ export class Search extends React.Component {
 
   actionPath() {
     const path = this.props.router.pathname;
-    return (path.includes('policies') || path.includes('requirements')) ? path : '/requirements/';
+    return path.includes('policies') || path.includes('requirements') ? path : '/requirements/';
+  }
+
+  handleChange(e) {
+    const { value } = e.target;
+    this.setState({ term: value });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const { router } = this.props;
+    const { term } = this.state;
+    const pathname = this.actionPath();
+    const query = { [this.inputName()]: term };
```

Comment:
I think we're losing the hidden fields here. The scenario we want to allow:
* User has filtered the requirements/policies by a topic (or multiple)
* User types something in the search bar at the top of the page
* Results should be _refined_ in the sense that the existing topics should remain.

Only annoying bit here is that we want to reset the page number.

Reply:
Ah, got it now, thanks for the explanation. Will fix and add a test case